test/TEST-13-NSPAWN-SMOKE/test.sh

```diff
@@ -4,6 +4,11 @@
 set -e
 TEST_DESCRIPTION="systemd-nspawn smoke test"
 TEST_NO_NSPAWN=1
+# if user namespaces are supported on the host, enable them
+# in the VM/container as well
```

Comment:
I'd add that it's CentOS-specific. Maybe it'd even make sense to guard it with something like `if $LOOKS_LIKE_CENTOS`, but I'm not sure.

Reply:
Guarding this part definitely makes sense (imo).

Reply:
In theory, it's possible to run any kernel with `KERNEL_BIN=` so technically we can't use this check to make sure that the kernel that is about to be booted supports user namespaces. Maybe it would be better to remove this part and pass `KERNEL_APPEND` explicitly on CentOS CI. What do you think?

Reply:
I already override `KERNEL_BIN` and `INITRD`, so I see no issues here. Will drop this part completely then, thanks!